jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -4934,7 +4939,7 @@
 						}
 					],
 					"request": {
-						"url": "{{url}}/v1.0/resources/HEALTH CHECK 4.0/HEALTH-CHECK-4.0/hct.xml",
+						"url": "{{url}}/v1.0/resources/{{HealthCheckGroup}}/{{HealthCheckApp}}/hct.xml",
```

Comment:
If we're trying to remove dependencies on CTP health check we should not be replacing hard coded values with variables, but instead using postman deployed groups, apps, JVMs and web servers. 

We made the changes in some places, so why not here as well?